src/components/Tooltip/Tooltip.js

```diff
@@ -26,31 +26,110 @@ const Wrapper = styled.div`
 
 const Tooltip = ({
   children,
-  text,
-  delay,
   className,
+  disableFocusListener,
+  disableMouseListener,
+  enterDelay,
+  leaveDelay,
+  onBlur,
+  onClose,
+  onFocus,
+  onMouseEnter,
+  onMouseLeave,
+  onOpen,
   style,
+  testId,
+  text,
   ...otherProps
 }) => {
   const [show, setShow] = useState(false);
-  const [delayTimer, setDelayTimer] = useState(null);
+  const [openTimer, setOpenTimer] = useState(null);
+  const [closeTimer, setCloseTimer] = useState(null);
+
+  const useFocus = !disableFocusListener;
```

Comment:
Declaring variables with the _use_ prefix might cause some mental overhead when reading the code since we usually have it for react hooks. What do you think about renaming it for something like _isUsingFocus_? _(the same can be applied for `useMouse`)_

Reply:
100%. much prefer the `is` prefix.

Reply:
updated in https://github.com/arturbien/React95/pull/99/commits/82e7600b527fd3f9a89548b1231816ec50a3680c